t/mojo/promise.t

```diff
@@ -181,6 +181,20 @@ Mojo::IOLoop->one_tick;
 is_deeply \@results, [], 'promises not resolved';
 is_deeply \@errors, ['second'], 'promise rejected';
 
+# Timeout
+(@errors, @results) = @_;
+$promise = Mojo::Promise->timeout(0.25 => 'Timeout1');
+$promise2 = Mojo::Promise->new->timeout(0.025 => 'Timeout2');
+$promise3
+  = Mojo::Promise->race($promise, $promise2)->then(sub { @results = @_ })
+  ->catch(sub { @errors = @_ })->wait;
+is_deeply \@results, [], 'promises not resolved';
+is_deeply \@errors, ['Timeout2'], 'promise rejected';
+
```

Comment:
Extra newline.

Reply:
I added a newline to indicate a new test case. Should I remove the newline or add a new heading, such as `# Timeout with default catch message`?

Reply:
Yes.

Reply:
My question was "should I remove the newline OR add a new heading". I'm not sure what you're answering "Yes" to.

Reply:
I added a new heading, since it's a new test case. I think that makes sense.